### ISSUE TITLE
--xlsx option is generating csv files and --csv excel file

### ISSUE THREAD

**Issue by yolile** [open-contracting/spoonbill#68]
When using the --xlsx option only CSV files are generated and when using the --csv one only an excel file is generated.

Additionally, the command help says:

````
  --xlsx               Path to result xlsx file
  --csv                Path to directory for output csv files
````

But when trying to set a path, eg `spoonbill --csv data file.json` I get an error:

`Error: Got unexpected extra argument`

**Comment by yolile** [comment]
may be closed by #63

**Comment by stasivoleh** [comment]
Checked --csv, --xlsx - **PASS**
